assets/README.md

```diff
@@ -0,0 +1,150 @@
+![MuseScore Logo](resources/mu.svg)
+
+MuseScore Assets
+================
+
+This is where all assets are stored as SVG (Scalable Vector Graphics) files.
+These source files are processed by various command line tools, including
+[Inkscape], to produce more SVGs, as well as raster images (PNG) and icons
+(ICO, ICNS) at build time. There is an option to download pre-generated assets.
+
+[Inkscape]: https://inkscape.org/ "Inkscape open source vector graphics editor"
+
+## Why compile assets?
+
+Why not just commit PNG & ICO files into the repository?
+
+- Binary files are not stored efficiently by `git`.
+- The GPL requires all source files to be distributed.
+- Building assets saves somebody from having to generate the files manually.
+  - Reduced potential for mistakes.
+  - Ensures assets match on all platforms.
+
+## Structure
+
+- __Resources:__ Textures and basic shapes that are used to make other assets
+- __Brand:__ Logos
+- __Icons:__ App icons and document filetype/mimetype icons
+- __Splash:__ Splash screen images displayed at MuseScore's startup
+- _Glyphs:_ in-app icons for MuseScore's buttons and menus are currently
+  stored in [../mscore/data/icons](../mscore/data/icons).
+
+## Build
+
+The CMake build system is used to generate the assets. The build rules are
+stored in `CMakeList.txt` and the `*.cmake` files.
+
+The assets are built automatically as part of MuseScore's build, but it is
+possible to build the assets separately (e.g. for testing), like this:
+
+```bash
+cd /path/to/MuseScore/assets/
+mkdir build
+cd build
+cmake ..
+make
+```
+
+## Dependencies
+
+### Programs
+
+The first time you run CMake you are likely to get an error message saying the
+build has failed due to a missing dependency (`Not found: [name of program]`).
+Simply install the required program and try again.
+
+### Fonts
+
+Text will not be rendered correctly unless you have the right fonts installed.
+Currently, the required fonts are:
+
+- [Raleway] (used in the splash screens)
+- [Roboto] (used in the file icons and splash screens)
+
+[Raleway]: https://fonts.google.com/specimen/Raleway
+[Roboto]: https://fonts.google.com/specimen/Roboto
```

Comment:
Roboto is Apache 2 licenced and therefore incompatible with the GPLv2.

Raleway is SIL OFL, which is fine for font embedding.

Reply:
Is there any particular reason why Musescore still uses V2? V3 would greatly help in dependency compatibility.

Reply:
GNU also seems to lay out some pretty compelling reasons to switch [here](https://www.gnu.org/licenses/rms-why-gplv3.en.html)

Reply:
@mirabilos, well spotted!

@liamrosenfeld, there have been discussions about switching from GPLv2 to v3 but it can't happen overnight. (There are also good [arguments to avoid v3](https://www.youtube.com/watch?v=PaKIZ7gJlRU), so it's not entirely clear-cut.)